README.md

```diff
@@ -58,6 +58,7 @@ processor.call(text)
 :language_aliases    - Alias table for some language names. (Hash)
 :rule                - Sanitization rule table. (Hash)
 :script              - A flag to allow to embed script element. (Boolean)
+:hostname            - FQDN to check whether or not each URL of `href` attributes is external site. (String)
```

Comment:
It's a bit strange that this description says FQDN (_Fully_ Qualified Domain Name) but the actual logic treat URLs with subdomain as internal links.